config/jest.config.json

```diff
@@ -11,9 +11,9 @@
   "collectCoverage": true,
   "coverageThreshold": {
     "global": {
-      "branches": 80,
-      "functions": 80,
-      "lines": 80,
+      "branches": 50,
+      "functions": 50,
+      "lines": 50,
```

Comment:
Also ja, 50% sind nur gefordert, aber wir haben ja auch 80% overall

Reply:
In den meisten Dateien haben wir mehr als 80%, aber nicht in allen. So sieht dann mehr grün aus :D